daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Pretty.scala

```diff
@@ -153,87 +153,34 @@ private[lf] object Pretty {
-            intercalate(comma + space, required.map(prettyParty))
-
-        case (
-            nodeId,
-            FailedAuthorization.MaintainersNotSubsetOfSignatories(
-              templateId @ _,
-              optLoc @ _,
-              signatories,
-              maintainers,
-            ),
-            ) =>
-          str(nodeId) & text(": all the maintainers:") &
-            intercalate(comma + space, maintainers.map(prettyParty)) +
-              text(", need to be signatories:") &
-            intercalate(comma + space, signatories.map(prettyParty))
-
-        case (
-            nodeId,
-            FailedAuthorization
-              .FetchMissingAuthorization(templateId @ _, optLoc @ _, authorizing, stakeholders),
-            ) =>
-          str(nodeId) & text(": missing authorization for fetch, authorizing parties:") &
-            intercalate(comma + space, authorizing.map(prettyParty)) +
-              text(", at least one of the following parties need to authorize:") &
-            intercalate(comma + space, stakeholders.map(prettyParty))
-
-        case (
-            nodeId,
-            FailedAuthorization.ExerciseMissingAuthorization(
-              templateId @ _,
-              choiceId @ _,
-              optLoc @ _,
-              authorizing,
-              required,
-            ),
-            ) =>
-          str(nodeId) & text(": missing authorization for exercise, authorizing parties:") &
-            intercalate(comma + space, authorizing.map(prettyParty)) +
-              text(", exactly the following parties need to authorize::") &
-            intercalate(comma + space, required.map(prettyParty))
-
-        case (
-            nodeId,
-            FailedAuthorization.ActorMismatch(templateId @ _, choiceId @ _, optLoc @ _, actors)) =>
-          str(nodeId) + text(": actor mismatch, given actors:") &
-            intercalate(comma + space, actors.map(prettyParty))
-
-        case (nodeId, FailedAuthorization.NoSignatories(templateId, optLoc @ _)) =>
-          str(nodeId) + text(s": $templateId missing signatories")
-
-        case (nodeId, FailedAuthorization.NoControllers(templateId, choiceId, optLoc @ _)) =>
-          str(nodeId) + text(s": $templateId $choiceId missing controllers")
-
-        case (
-            nodeId,
-            FailedAuthorization.LookupByKeyMissingAuthorization(
-              templateId @ _,
-              optLoc @ _,
-              maintainers,
-              authorizingParties,
-            ),
-            ) =>
-          str(nodeId) + text(": missing authorization for lookup by key, authorizing parties:") &
-            intercalate(comma + space, authorizingParties.map(prettyParty)) +
-              text(" are not a superset of maintainers:") &
-            intercalate(comma + space, maintainers.map(prettyParty))
-      },
-    )
+  private def authorizationErrors(failures: Map[NodeId, FailedAuthorization]): String = {
```

Comment:
Why can't we keep or at least adapt the old pretty printer for now?

Reply:
see reply above